quadriga/pom.xml

```diff
@@ -39,11 +39,14 @@
         <mongo.database.authdb>admin</mongo.database.authdb>
 
         <neo4j.url>http://neo4j:quadriga@localhost:7474/</neo4j.url>
-        <neo4j.database>quadriga</neo4j.database>
+        <neo4j.database>quadriga2</neo4j.database>
```

Comment:
please don't change this (database url above is fine as it would just set a sensible default, but this should not be changed).